scss/abstracts/_colors.scss

```diff
@@ -0,0 +1,72 @@
+// ------------------------------------------------------- COLORS ------------------------------------------------------
+// ----------------------------------------------------- ONS COLORS ----------------------------------------------------
+
+// --- ONS General palette ---
+$spring-green: #A8BD3A;
+$aqua-teal: #00A3A6;
+$sky-blue: #27A0CC;
+$ocean-blue: #206095;
+$night-blue: #003C57;
+
+// --- Census ---
+$indigo-blue: #3C388E;
+$plum-purple: #902082;
+$flamingo-pink: #DF0667;
+
+// --- Supporting palette ---
+$ruby-red: #D0021B;
+$jaffa-orange: #FE781F;
+$sun-yellow: #FBC900;
+$neon-yellow: #F0F762;
+$leaf-green: #0F8243;
+
+// --- Grays ---
+$black: #222;
+$grey1: #414042;
+$grey2: #707071;
+$grey3: #BCBCBD;
+$grey4: #E2E2E3;
+$grey5: #F5F5F6;
+$white: #FFF;
+
+
+// --- Tints ---
+// Allowed tint percentages
+$tints: (70, 40, 10);
+// colours to be used (currently all listed in colour palettes except greys)
+$tintable-colors: (
```

Comment:
Is there any specific reason we use a map here instead of a list?

It'd mean instead of setting K/V pairs we can just have `$tintable-colors: $spring-green, $aqua-teal, $sky-blue;` etc. The `tint` function would also need to change so we check `index` instead of using `map-has-keys`. 

So something like: `@if index($tints, $percentage) and index($tintable-colors, $color)` in the function

Appreciate I might be overlooking something, but thought I'd raise it nonetheless

Reply:
Tis now a list, and a good point to raise